tests/testthat/test-subset_obs.R

```diff
@@ -0,0 +1,93 @@
+# Daily data
+obs_daily <- enw_example("nowcast")$latest[[1]]
+# Mock weekly data
+obs_weekly <- data.table(
```

Comment:
This is very handy - nice. We could move this to test/testhat/setup.R as its repeated and might be useful elsewhere?